### ISSUE TITLE
Workflow: Feature branches forus-frontend

### ISSUE THREAD

**Issue by maxvisser** [teamforus/forus#355]
## Context/goal: 
issue about workflow Yaroslav; starting to use feature branches! great! :) You made only a small error.
What you now did was not checkout develop after creating the feature branch and commiting/pushing it. 

These are the command you need to use:
```
git checkout develop

**work on feature**

git add . 

git commit -m "fix #111" 

git branch feature-name-of-fix

git push --set-upstream origin feature-name-of-fix

# open a pull request for the feature branch you've just pushed 
git pull-request  // type fixed issue #111 

git checkout develop // what you forgot
'
**repeat work on different feature**

git add . 

git commit -m "fix #222" 

etc...
```

You can now delete the branches:
feature-information-provider
feature-wrong-price-old-price
feature-hide-shopkeeper-email 

as they are already on feature-annuleren-button (because of the mistake of making branches from your feature branch)

basically this scenario:
![image](https://user-images.githubusercontent.com/10818702/48441813-c157e200-e78c-11e8-8dcd-f61afa7c56a3.png)

while I want this:
![image](https://user-images.githubusercontent.com/10818702/48442063-6377ca00-e78d-11e8-969a-65f2e7e6bb8c.png)

## aditional info
if pull-request doesnt work please try installing hub https://hub.github.com/

**Comment by ateamcms** [comment]
Yes, thank you, I made a mistake with the setting of PhpStorm, in the future I will try to do it right but if you will see something wrong, please tell me.

**Comment by maxvisser** [comment]
no problem, will do :) i close this issue for now.
